core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -663,61 +840,72 @@ export class Matrix3d implements BeJSONFunctions {
-   * * With invert === true the matrix is transposed so that `matrix.multiply(viewVector` maps the "in view" vector to a world vector.
+   * * Default is TOP view (`local X = world X`, `local Y = world Y`, `local Z = world Z`).
+   * * To change view from the TOP to one of the other 7 standard views, we need to multiply "world data" to
+   * the corresponding matrix1 provided by `createStandardWorldToView(index, false)` and then
+   * `matrix1.multiply(world data)` will returns "local data".
+   * * To change view back to the TOP, we need to multiply "local data" to the corresponding matrix2 provided
+   * by `createStandardWorldToView(index, true)` and then `matrix2.multiply(local data)` will returns "world data".
    *
-   * @param index standard view index `StandardViewIndex.Top, Bottom, LEft, Right, Front, Back, Iso, LeftIso`
-   * @param invert if false (default), the returned Matrix3d "projects" world vectors into XY view vectors.  If true, it is inverted to map view vectors to world.
+   * @param index standard view index `StandardViewIndex.Top, Bottom, Left, Right, Front, Back, Iso, RightIso`
+   * @param invert if false (default), the return matrix is world to local (view) and if true, the the return
+   * matrix is local (view) to world.
    * @param result optional result.
    */
   public static createStandardWorldToView(index: StandardViewIndex, invert: boolean = false, result?: Matrix3d): Matrix3d {
     switch (index) {
-
+      // start with TOP view, ccw rotation by 180 degrees around X
       case StandardViewIndex.Bottom:
         result = Matrix3d.createRowValues(
           1, 0, 0,
           0, -1, 0,
           0, 0, -1);
         break;
+      // start with TOP view, ccw rotation by -90 degrees around X and by 90 degrees around Z
       case StandardViewIndex.Left:
         result = Matrix3d.createRowValues(
           0, -1, 0,
           0, 0, 1,
           -1, 0, 0);
         break;
+      // start with TOP view, ccw rotation by -90 degrees around X and by -90 degrees around Z
       case StandardViewIndex.Right:
         result = Matrix3d.createRowValues(
           0, 1, 0,
           0, 0, 1,
           1, 0, 0);
         break;
-      case StandardViewIndex.Front: // 0-based 4
+      // start with TOP view, ccw rotation by -90 degrees around X
+      case StandardViewIndex.Front:
         result = Matrix3d.createRowValues(
           1, 0, 0,
           0, 0, 1,
           0, -1, 0);
         break;
-      case StandardViewIndex.Back: // 0-based 5
+      // start with TOP view, ccw rotation by -90 degrees around X and by 180 degrees around Z
+      case StandardViewIndex.Back:
         result = Matrix3d.createRowValues(
           -1, 0, 0,
           0, 0, 1,
           0, 1, 0);
         break;
       case StandardViewIndex.Iso:
+        // start with FRONT view, ccw rotation by -45 degrees around Y and by 35.264 degrees around X
```

Comment:
Missing comment for RightIso case:
`// start with FRONT view, ccw rotation by 45 degrees around Y and by 35.264 degrees around X`

Reply:
yes, I already noticed that and added to the next PR